release-notes/2025-09/README.md

```diff
@@ -0,0 +1,164 @@
+# SystemLink Enterprise 2025-09 Release Notes
+
+The 2025-09 release for SystemLink Enterprise has been
+published to <https://downloads.artifacts.ni.com>. This update includes new
+features, bug fixes, and security updates. Work with your account representative
+to obtain credentials to access these artifacts. If you are not upgrading from
+the previous release, refer to past release notes to ensure you have addressed
+all required configuration changes.
+
+## Upgrading from the 2025-08 to the 2025-09
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
+
+- View all test plans scheduled for a specific system in the System Details page. For more information, refer to [Viewing Scheduled Test Plans](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/viewing-scheduled-test-plans.html).
+- Use the Schedule page to search and filter for fixtures.
+- Customize test plan workflows to match the processes of your organization. For more information, refer to the [Test Plan Operations Example page in GitHub](https://github.com/ni/systemlink-enterprise-examples/tree/main/Test%20Plan%20Operations%20Examples).
+- Manage system keys to resolve connection and security issues.
+- Use the Assets data source in dashboards to display the total count of returned assets in a query.
+- Compatibility with Grafana 11.
+- `helium-fileingestionservices:1.19.*`
+  - Introduced the chunked file upload API for upload sessions.
+  - Bypass any ingress controller limitations or IT restrictions that prevent the uploading of files larger than 1 GB.
+  - Available only on the back-end side. As of this release, the Files Web Application does not leverage this API.
+- `workorder:0.20.*`
+  - Introduced new custom workflows. This includes the `wokflowSnapshot` field on test plans and the `workflowId` on test plans and test plan templates.
+  - Discontinued beta feature of `workflow` field on test plans.
+  - Discontinued support of test plans that use the `workflow` field and custom `executionActions`. To achieve similar behavior, you can create a workflow using the workflows API in the Work Order service. You then need to assign the `id` of the workflow to the `workflowId` field of the new test plans or test plan templates.
+  - Compatibility between `executionActions` of test plans and test plan templates with the new workflows.
+
+## Helm Chart Breaking Changes
```

Comment:
No breaking changes! If so we should keep this section, but state that there are no breaking changes.